### PR TITLE
pritunl-ssh: init at 1.0.1674.4

### PR DESCRIPTION
**Pull request by Thunderbottom**
pritunl-ssh is a small Python script used to retrieve the SSH certificate from the Pritunl Zero server.

Homepage: https://zero.pritunl.com/
GitHub: https://github.com/pritunl/pritunl-zero-client

Signed-off-by: Chinmay D. Pai <chinmaydpai@gmail.com>

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -9363,6 +9363,16 @@
     githubId = 1391883;
     name = "Tom Hall";
   };
+  Thunderbottom = {
```
Comment (rmcgibbo) [inline maintainers/maintainer-list.nix:9366]: Can you split this into a separate commit with a message like 'maintainers: add Thunderbottom'

File: pkgs/tools/networking/pritunl-ssh/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  version = "1.0.1674.4";
+
+  src = fetchFromGitHub {
+    owner = "pritunl";
+    repo = "pritunl-zero-client";
+    rev = "1.0.1674.4";
+    sha256 = "07z60lipbwm0p7s2bxcij21jid8w4nyh6xk2qq5qdm4acq4k1i88";
+  };
+
+  patchPhase = ''
```
Comment (rmcgibbo) [inline pkgs/tools/networking/pritunl-ssh/default.nix:14]: Can you try using `patchShebangs` instead? This should be handled semi-automatically.

**Review by rmcgibbo — CHANGES_REQUESTED** [review]

File: pkgs/tools/networking/pritunl-ssh/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  pname   = "pritunl-ssh";
+  version = "1.0.1674.4";
+
+  src = fetchFromGitHub {
+    owner = "pritunl";
+    repo = "pritunl-zero-client";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/pritunl-ssh/default.nix:10]:
```suggestion
    rev = version;
```

File: pkgs/tools/networking/pritunl-ssh/default.nix (hunk 2 above)
Comment (Thunderbottom) [reply pkgs/tools/networking/pritunl-ssh/default.nix:14]: I am not sure if I am getting this wrong, or maybe it seems to not be working for me for some reason.

I've tried adding `patchShebangs ssh_client.py` to the `patchPhase`, and that seems to not be working (i.e. does not output anything during the `nix-build`). I've also tried replacing `patchPhase` with `postPatch`, containing:
```
  postPatch = ''
    patchShebangs ssh_client.py
    patchShebangs ssh_host_client.py
  '';
```

which at least outputs the following during the `nix-build`:

```
...
patching script interpreter paths in ssh_client.py
patching script interpreter paths in ssh_host_client.py
...
```

But when I check the binary in `result/bin`, it still has `#!/usr/bin/env python3` at the top.

Am I doing something wrong?

File: pkgs/tools/networking/pritunl-ssh/default.nix (hunk 3 above)
Comment (Thunderbottom) [reply pkgs/tools/networking/pritunl-ssh/default.nix]: fixed, thanks for spotting!

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (Thunderbottom) [reply maintainers/maintainer-list.nix:9366]: this is done, thanks for spotting!

File: pkgs/tools/networking/pritunl-ssh/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/tools/networking/pritunl-ssh/default.nix:14]: I think you need to add python3 to build inputs.

Comment (Thunderbottom) [reply pkgs/tools/networking/pritunl-ssh/default.nix]: that worked, thank you!

**Comment by SuperSandro2000** [comment]
Please squash all the package related commits together.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113112` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>pritunl-ssh</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

pritunl-ssh:
<details>


`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/networking/pritunl-ssh/default.nix:16:3:

```
   |
16 |   installPhase = ''
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md
</details>
